javascript-review.js

```diff
@@ -3,112 +3,203 @@
 // 1.
 
 // Create an object that has four properties. One property should be set so that the value is a number, another property set to a string, the third to an array, and the fourth to a function. This function should simply console.log the value of the first property.
+const objectOne = {
+  numVariable: 2,
+  strVariable: "haha!",
+  arrVariable: [1, 2, 3, 4],
+  display() {
+    console.log(this.numVariable);
+  }
+}
 
 // 2.
 
-// Write a constructor function (to create objects) that takes one argument. It should use that argument to set the value of the first attribute of the object. There should be a second attribute that stores the value ‘red’. 
+// Write a constructor function (to create objects) that takes one argument. It should use that argument to set the value of the first attribute of the object. There should be a second attribute that stores the value ‘red’.
+
+const fruitConstructor = (name) => {
+  return {
+    name: name,
+    color: "red"
+  }
```

Comment:
a constructor function needs to  use `this` and the `new` keyword, they're the two keys i was looking for here, your example is more along the lines of a factory function

```js

function Car(make) {
  this.make = make
  this.color = 'red'
}

const holden = new Car('holden')
```